src/NuGet.Core/NuGet.Protocol.Core.v3/HttpSource/HttpSource.cs

```diff
@@ -68,12 +69,14 @@ public HttpSource(PackageSource source, Func<Task<HttpHandlerResource>> messageH
             _baseUri = new Uri(source.Source);
             _messageHandlerFactory = messageHandlerFactory;
             _retryHandler = new HttpRetryHandler();
+            DownloadTimeout = TimeSpan.FromSeconds(60);
```

Comment:
put it inline with a property itself. Better define a "constant" DefaultDownloadTimeout for the initial value.